csharp/src/Ice/InputStream-Class.cs

```diff
@@ -74,17 +74,38 @@ public void IceStartSlice(string typeId)
         }
 
         /// <summary>Reads a class instance from the stream.</summary>
-        /// <returns>The class instance, or null.</returns>
-        public T? ReadClass<T>() where T : AnyClass
+        /// <returns>The class instance read from the stream.</returns>
+        public T ReadClass<T>() where T : AnyClass
```

Comment:
Could it just be `ReadOptionalClass<T>() ?? throw new InvalidDataException("read null...");`?

Reply:
Yes, fixed.